vms/event/services.py

```diff
@@ -12,6 +14,21 @@ def event_not_empty(event_id):
     return result
 
 
+def get_event_by_shift_id(shift_id):
```

Comment:
@vubo @tapasweni-pathak It may be semantics but I would prefer a docstring over a comment :)